composer.json

```diff
@@ -15,12 +15,12 @@
 		"php-stubs/wordpress-stubs": "^5.5",
 		"php-stubs/wordpress-globals": "^0.2.0",
 		"php-stubs/wp-cli-stubs": "^2.5",
-		"vimeo/psalm": "^4"
+		"vimeo/psalm": "^4 || ^5 || ^5@beta"
```

Comment:
Unsure of the feasability of this, but maybe we should try and configure GitHub Actions to test each version

Currently, `vimeo/psalm (5.0.0-beta1)` is being installed

https://github.com/humanmade/psalm-plugin-wordpress/actions/runs/3342040266/jobs/5533851486#step:3:79